runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingTabPanel.java

```diff
@@ -261,7 +261,7 @@ public void update()
 
 	private AsyncBufferedImage getPatchImage(CompostState compostState, boolean protected_)
 	{
-		return protected_ ? itemManager.getImage(ItemID.APPLES5) :
```

Comment:
?

Reply:
Ah yeah, so a couple reasons for this change:

1) The apple basket implies the protection is due to payment to a gardener, but with Geomancy we get additional data about protection state such as Hosidius favor, Weiss fire, Troll Stronghold roof, flower patch protection, etc. The shield is a more generic representation of 'protection' and applies to all those scenarios.

2) The basket sprite itself often has low contrast agains the crop sprites, so it can be hard to tell at a glance. The shield generally has much better contrast:

<img width="239" alt="Screenshot 2023-08-02 at 3 42 18 PM" src="https://github.com/runelite/runelite/assets/12495920/6b315910-5667-4a0f-998e-28a1c3449994">